Tests/AwesomeAssertions.Equivalency.Specs/BasicSpecs.cs

```diff
@@ -569,7 +569,7 @@ public void When_asserting_equivalence_of_objects_including_enumerables_it_shoul
 
         // Assert
         act.Should().Throw<XunitException>().WithMessage(
-            @"Expected property record.Member1* to be*""different"" with a length of 9, but*"""" has a length of 0*");
+            """Expected property record.Member1 to be the same string* differ at index 0:*(actual)*""*"different"*(expected).""");
```

Comment:
Don't forget to add an asterisk `*` at the very end (except you add the full remaining message to the assertion.

```suggestion
            """Expected property record.Member1 to be the same string* differ at index 0:*(actual)*""*"different"*(expected).""*");
```